internal/wrapper/wrap_handler.go

```diff
@@ -133,10 +132,17 @@ func validateHandler(handler interface{}) error {
 	return nil
 }
 
-func callHandler(ctx context.Context, msg json.RawMessage, handler interface{}) (interface{}, error) {
-	ev, err := unmarshalEventForHandler(msg, handler)
-	if err != nil {
-		return nil, err
+func callHandler(ctx context.Context, msg json.RawMessage, handler interface{}) (response interface{}, errResponse error) {
+	defer func() {
+		if r := recover(); r != nil {
+			response = nil
+			errResponse = fmt.Errorf("Panic: %v", r)
+		}
```

Comment:
I'm concerned that since we do not re-panic, we're actually changing how a panicking function functions.

Reply:
tbh, I have the very same concern. 
I think the alternative would be to re-panic after we send the trace. 
```go
defer func() {
        if r := recover(); r != nil {
            // send the trace somehow 
            panic(r)
        }
    }()
```